cmd/controller/controller-runtime-main.go

```diff
@@ -42,12 +45,24 @@ type SchemeFunc func(*runtime.Scheme) error
 // ProvideFunc adds a controller to a Manager.
 type ProvideFunc func(manager.Manager) (controller.Controller, error)
 
+var knownProvideControllers = map[string]ProvideFunc{
```

Comment:
Should this be an exported variable? Then it can be documented with an explanation of the configmap and its format.